lib/assessment.md

```diff
@@ -138,7 +160,10 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+```CSS
+the div class would be placed at the top left corner of the webpage.
+.divOne would be placed at the top right corner of the webpage.
+.divFour would be placed at the top left corner of the webpage seperate from the other divs.
```

Comment:
I think you may need a little more review of this. Try copying the HTML and CSS here into CodePen and see what happens - can you explain why the divs are laid out the way they are?